ppl/zqd/core.go

```diff
@@ -136,6 +141,13 @@ func (c *Core) addWorkerRoutes() {
 	c.handle("/worker", handleWorker).Methods("POST")
 }
 
+func (c *Core) addRecruiterRoutes() {
+	c.handle("/deregister", handleDeregister).Methods("POST")
+	c.handle("/recruit", handleRecruit).Methods("POST")
+	c.handle("/register", handleRegister).Methods("POST")
+	c.handle("/unreserve", handleUnreserve).Methods("POST")
```

Comment:
Instead of putting these commands in the root namespace, can prefix this commands with `/recruiter`?

Reply:
Or maybe `/workers`?

Reply:
OK, I will make that change